### PR TITLE
add plugin "blame"

### PR DESCRIPTION
**Pull request by shpwrck**
Plugin has been verified.

I don't know if this will make it into the core code eventually, but I believe having it in krew will provide useful analytics to see if it should be.

### CODE REVIEW

**Comment by shpwrck** [comment]
"I signed it"

File: plugins/blame.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      matchLabels:
+        os: darwin
+        arch: amd64
+  shortDescription: Annotate each line in the given resource's YAML with information from the managedFields to show who last modified the field.
+  homepage: https://github.com/knight42/kubectl-blame
+  caveats: |
+      As long as the field '.metadata.managedFields' of the given resource is set properly, this command is able to display the manager of each field.
+
+      Usage:
+        blame TYPE[.VERSION][.GROUP] NAME
```
Comment (ahmetb) [inline plugins/blame.yaml:38]: No need for usage string here.

File: plugins/blame.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      - from: ./kubectl-blame
+        to: blame
+    selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+  shortDescription: Annotate each line in the given resource's YAML with information from the managedFields to show who last modified the field.
+  homepage: https://github.com/knight42/kubectl-blame
+  caveats: |
+      As long as the field '.metadata.managedFields' of the given resource is set properly, this command is able to display the manager of each field.
```
Comment (ahmetb) [inline plugins/blame.yaml:35]: Wrap at 80 chars please.

File: plugins/blame.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    files:
+      - from: ./LICENSE
+        to: .
+      - from: ./kubectl-blame
+        to: blame
+    selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+  shortDescription: Annotate each line in the given resource's YAML with information from the managedFields to show who last modified the field.
```
Comment (ahmetb) [inline plugins/blame.yaml:32]: This is not short at all. It won't show most of it. Please look at other plugins.

File: plugins/blame.yaml (hunk 2 above)
Comment (shpwrck) [reply plugins/blame.yaml]: Wrapped

File: plugins/blame.yaml (hunk 1 above)
Comment (shpwrck) [reply plugins/blame.yaml]: Removed Usage

**Comment by ahmetb** [comment]
You can't mention people like @ in your commits. It's spammy. Please amend history.

**Comment by shpwrck** [comment]
Sorry, that was not my intention. I was using it as shorthand for "at". Will fix.

**Comment by shpwrck** [comment]
I removed the "at" in the previous commit. This should be ready for approval.

**Comment by ahmetb** [comment]
/hold
this plugin is not importing auth plugins.

Please check out the [Best Practices](https://krew.sigs.k8s.io/docs/developer-guide/develop/best-practices/#auth-plugins) guide and the [Checklist](https://krew.sigs.k8s.io/docs/developer-guide/release/new-plugin/).

**Comment by ahmetb** [comment]
I noted two more issues:

1. You don't seem to be the author of the plugin repository. Are you co-authoring/maintaining this repo? We usually expect repo owners are aware of their plugin distributed on Krew so that they can set up further release automation and typically we contact the owner of the repo for any issues with the plugin manifest.
2. This plugin seems to require a certain Kubernetes version or higher. This is a big caveat, let's add it to `caveats:` if that's the case.

**Comment by shpwrck** [comment]
1. I am not a maintainer, but I've spoken with Jian Zeng on this regard.
    I helped restructure the release output so that we could make this a plugin.

2. I actually just mentioned in my CNCN.io show today that I should add a Caveat in regards to requiring a specific version of kubernetes. I will add it.

3. I will admit this is an odd request. I'm effectively an intermediary. But I want to see this go through. I've opened [this issue](https://github.com/knight42/kubectl-blame/issues/2) to link the two audiences.

File: plugins/blame.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    files:
+      - from: ./LICENSE
+        to: .
+      - from: ./kubectl-blame
+        to: blame
+    selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+  shortDescription: Show who last modified a field.
```
Comment (ahmetb) [inline plugins/blame.yaml:32]: nit: no trailing period please

Comment (shpwrck) [reply plugins/blame.yaml]: Updated

**Comment by ahmetb** [comment]
Yeah I'm happy to move forward as long as the actual maintainer is supportive as setting up stuff like release automation requires their permission.

**Comment by ahmetb** [comment]
Let me know with @ when you make progress getting the author to be on board with this.
I still don't see auth plugins updated so I had to create a non-GKE cluster to test it.
A new version would be nice before we accept.

**Comment by corneliusweig** [comment]
Really nice idea! Looking forward to see this land in `krew`! :rocket:

**Pull request closed without merging**